contextualized/easy/ContextualizedNetworks.py

```diff
@@ -82,7 +82,7 @@ def measure_mses(self, C, X, individual_preds=False):
             for j in range(X.shape[-1]):
                 tiled_xi = np.array([X[:, i] for _ in range(len(betas))])
                 tiled_xj = np.array([X[:, j] for _ in range(len(betas))])
-                residuals = tiled_xi - betas[:, :, i, j] * tiled_xj - mus[:, :, i, j]
+                residuals = tiled_xi - betas[:, :, i, j] * tiled_xj + mus[:, :, i, j]
```

Comment:
why +?

Reply:
This file was modified from the networks benchmarking branch version, I'm not sure

Reply:
- is correct, this bug was fixed in #171.